lxd-stophook-wrapper/lxd

```diff
@@ -0,0 +1,8 @@
+#!/bin/sh
+# Use exec so that this script process is replaced.
+# This avoids polluting the process tree with this wrapper script.
+if [ "$1" = "callhook" ]; then
+    exec /snap/lxd/current/bin/lxd-user "$@"
+else
+    exec lxd "$@"
+fi
```

Comment:
Just a question of style but elsewhere, we put the second `exec` outside of any `if/else` clause as the one inside the `if` is terminal anyway. https://github.com/canonical/lxd-pkg-snap/blob/latest-edge/snapcraft/wrappers/run-host#L10-L14 is one such example.

Reply:
yeah lets do that